apps/voting/app/src/components/VoteCard/VoteCard.js

```diff
@@ -110,10 +144,16 @@ function VoteCard({ vote, onOpen }) {
           margin-top: ${2 * GU}px;
         `}
       >
-        {open ? (
-          <Timer end={endDate} maxUnits={4} />
+        {disputableStatus === VOTE_STATUS_ACTIVE ? (
+          open ? (
+            <Timer end={endDate} maxUnits={4} />
+          ) : (
+            <VoteStatus vote={vote} />
+          )
         ) : (
-          <VoteStatus vote={vote} />
+          disputableStatus && (
+            <DisputableStatusLabel status={disputableStatus} />
+          )
```

Comment:
Same thing here, if we always have the status (and it's not being delivered async or something) then no need to check :)